File: location-based-services/jdk-1.8/LocationBasedServices/app/src/main/java/com/example/anhkhue/locationbasedservices/MainActivity.java
package com.example.anhkhue.locationbasedservices;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    private static final String TAG = MainActivity.class.getCanonicalName();

    private Button btnOpen;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        Log.d(TAG, "Create.");

        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnOpen = findViewById(R.id.btnOpen);

        btnOpen.setOnClickListener(view -> {
            Intent intent = new Intent(getApplicationContext(), MapsActivity.class);
            startActivity(intent);
        });
    }
}

File: location-based-services/kotlin/LocationBasedServices/app/src/main/java/com/example/anhkhue/locationbasedservices/MainActivity.kt
package com.example.anhkhue.locationbasedservices

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Button

class MainActivity : AppCompatActivity() {
    companion object {
        const val TAG = "MainActivity"
    }

    private lateinit var btnOpen: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        Log.i(TAG, "On Create")

        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnOpen = findViewById(R.id.btnOpen)
        btnOpen.setOnClickListener {
            val intent = Intent(this, MapsActivity::class.java)
            startActivity(intent)
        }
    }
}
